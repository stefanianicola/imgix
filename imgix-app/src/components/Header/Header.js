import React from 'react';
import './Header.scss';
import { Link } from 'react-router-dom';
import { Navbar, Col } from 'react-bootstrap';
import 'bootstrap/dist/css/bootstrap.min.css';



function Header() {


    return (
        <Col xs={12} className="p-0 head">
            <Navbar expand="md">
                <Col xs={12} className="wrap-logo">
                    <Navbar.Brand >
                        <Link to="/">
                            <img
                                src="https://assets.imgix.net/presskit/imgix-presskit.pdf?page=3&fm=png&w=320&dpr=2"
                                width="auto"
                                height="30"
                                className="d-inline-block align-top"
                                alt=""
                            />
                        </Link>
                    </Navbar.Brand>
                </Col>
            </Navbar>
        </Col>
    )
}

export default Header;