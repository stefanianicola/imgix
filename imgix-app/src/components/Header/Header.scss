.head {
    background-color: #fff; //#f0743f;
    .wrap-logo {
        text-align: center;
    }
}
