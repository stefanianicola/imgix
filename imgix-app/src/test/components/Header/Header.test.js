import React from 'react';
import '@testing-library/jest-dom';
import { shallow } from 'enzyme';
import Header from '../../../components/Header/Header'

describe('Header component', () => {


    const wrapper = shallow(<Header />);
    it('should be create the component', () => {
        expect(wrapper).toMatchSnapshot();
    });

})