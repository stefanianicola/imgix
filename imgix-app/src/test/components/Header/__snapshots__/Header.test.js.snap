// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component should be create the component 1`] = `
<Col
  className="p-0 head"
  xs={12}
>
  <Navbar
    collapseOnSelect={false}
    expand="md"
    variant="light"
  >
    <Col
      className="wrap-logo"
      xs={12}
    >
      <NavbarBrand>
        <Link
          to="/"
        >
          <img
            alt=""
            className="d-inline-block align-top"
            height="30"
            src="https://assets.imgix.net/presskit/imgix-presskit.pdf?page=3&fm=png&w=320&dpr=2"
            width="auto"
          />
        </Link>
      </NavbarBrand>
    </Col>
  </Navbar>
</Col>
`;
